apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/nodes/SignUpBlockNode/SignUpBlockNode.tsx

```diff
@@ -1,5 +1,5 @@
-import { useTranslation } from 'next-i18next'
 import { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
```

Comment:
rip, this needs to be changed back
```suggestion
import { useTranslation } from 'next-i18next'
import { ReactElement } from 'react'
```

Reply:
The react version is restricted in eslint. Are we still supposed to use the react version? If so, do we need to revert the rule?

Reply:
No, the correct import is from next. This was only a recent change so thats why some files are still using the react one. 

I think when I asked you to revert, it was before your actual signupblock tests when in. 
We basically want no changes in this file.

Reply:
Ohhhh I understand now haha I forgot we reverted already. My bad